tooling/image-updater/README.md

```diff
@@ -0,0 +1,63 @@
+# Image Updater
+
+Automatically fetches the latest image digests from container registries and updates ARO-HCP configuration files.
+
+## Usage
+
+```bash
+# Update all images
+make update
+
```

Comment:
We can add a table here listing the images that will be updated since some images cannot be updated currently such as ACM/MCE/CS.